app/tests/spec/views/settings/communication_preferences.js

```diff
@@ -31,6 +32,7 @@ define(function (require, exports, module) {
     var notifier;
     var preferencesUrl = 'https://marketing.preferences.com/user/user-token';
     var relier;
+    let sentryMetrics;
```

Comment:
Yes.

Unless by "it" you mean the local reference to it, in which case no, I could pass in `new SentryMetrics()` directly to the `Metrics` constructor directly. I was only trying to follow the convention used everywhere else but I can change it if it makes you happy.